<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eliminar</class>
 <widget class="QWidget" name="Eliminar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelEliminar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bEliminarUnidad">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR RECORRIDO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bEliminarRecorrido">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR UNIDAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bEliminarConductor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRegresarEliminar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
